assets/sass/modules/_btns.scss

```diff
@@ -53,7 +53,13 @@ $od-secondary-button-action-text-color: #124a94;
 .call-request-button,
 .email-request-button {
   text-align: center;
-  @include max-width-text(130px);
+  width: 140px;
+
+  // This css will break the text into two lines if it's too long.
+  white-space: normal;
+  overflow: visible;
+  line-height: 1;
+  padding-top: 12px;
```

Comment:
I could update the `max-width-text` mixin itself as it is not used anywhere else. But, I didn't do it for these reasons -
* Every element that includes it in the future will have the text fall to next line.
* Here since this PR only addresses special cases of buttons-next-to-inputs, I'd keep them subjected only to those cases.
* Important to note `padding-top` here as I'm changing the `line-height`.
* Given a chance and time, I'd want to change the mark-up and css. Mark-up changes would be to - 1. Use button element instead of anchor tag that would take care of these special i18n long text scenarios mostly. 2. Create a parent element to enclose the input and the button and make it a `flex` box. This would give a lot more control over how we want it rendered.